website/project/views/register.py

```diff
@@ -151,10 +424,32 @@ def node_register_template_page_post(auth, node, **kwargs):
     schema = MetaSchema.find(
         Q('name', 'eq', template)
     ).sort('-schema_version')[0]
+
+    # Create the registration
     register = node.register_node(
         schema, auth, template, json.dumps(clean_data),
     )
 
+    if data['registrationChoice'] == 'Make registration public immediately':
```

Comment:
I agree with @samchrisinger; don't use description text for the options' values. Create a mapping on the client-side of the valid values to their human-readable representations

```js
self.REGISTRATION_OPTIONS = [
    {value: 'immediate', description: 'Make registration public immediately.'
    //...
]
```

See http://knockoutjs.com/documentation/options-binding.html on how to correctly bind objects like this to a select element.